packages/x-data-grid-premium/src/hooks/features/chartsIntegration/useGridChartsIntegration.tsx

```diff
@@ -195,17 +193,20 @@ export const useGridChartsIntegration = (
       }
 
       const columns = gridColumnLookupSelector(apiRef);
-      const unwrappedColumnGroupingModel = gridColumnGroupsUnwrappedModelSelector(apiRef);
+      const columnGroupPath = apiRef.current.getColumnGroupPath(field);
+      const columnGroupLookup = apiRef.current.getAllGroupDetails();
```

Comment:
Nitpick: The API methods just shadow these selectors, how about directly using them. I have an impression we want to get rid of such API methods in favor of direct selector usages.
```suggestion
      const columnGroupPath = gridColumnGroupsUnwrappedModelSelector(apiRef)[field] ?? [];
      const columnGroupLookup = gridColumnGroupsLookupSelector(apiRef);
```